iron-list.html

```diff
@@ -780,8 +801,8 @@
      */
     updateViewportBoundaries: function() {
       var styles = window.getComputedStyle(this);
```

Comment:
@blasten 
Why do you get the styles for 'this', and not 'this.scrollTarget' ?

I have a scroll target (variation of paper-scroll-header-panel) with a padding  on top, and the first and last item calculation is only correct when using: 
`var styles = window.getComputedStyle(this.scrollTarget);`